pallets/claims/src/lib.rs

```diff
@@ -0,0 +1,207 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+use codec::{Decode, Encode};
+use frame_support::{
+	decl_error, decl_event, decl_module, decl_storage,
+	dispatch::DispatchResult,
+	ensure,
+	traits::{Currency, Get, Imbalance},
+	weights::{DispatchClass, Pays},
+};
+use frame_system::ensure_signed;
+use primitives::Balance;
+use sp_runtime::traits::Zero;
+use sp_std::prelude::*;
+use sp_std::vec::Vec;
+
+use frame_support::sp_runtime::traits::{DispatchInfoOf, SignedExtension};
+use frame_support::sp_runtime::transaction_validity::{
+	InvalidTransaction, TransactionValidity, TransactionValidityError, ValidTransaction,
+};
+use frame_support::traits::IsSubType;
+use sp_std::marker::PhantomData;
```

Comment:
Consolidate imports. Got multiple lines of the same package. You can use the ::{} syntax

Reply:
Also reorder pls